### PR TITLE
Implement new tcl-extension-file check (#259).

### PR DESCRIPTION
**Pull request by marxin**

### CODE REVIEW

**Comment by scarabeusiv** [comment]
Needs rebase.

**Comment by marxin** [comment]
> Needs rebase.

Done

File: test/test_files.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def test_sphinx_inv_files(tmpdir, package, filescheck):
     assert not len(output.results)
 
 
+<<<<<<< HEAD
```
Comment (scarabeusiv) [inline test/test_files.py]: Should not be here

File: rpmlint/checks/FilesCheck.py [hunk 2]
```diff
@@ -836,6 +837,9 @@ def check(self, pkg):
                     if pkg.grep(start_private_key_regex, f):
                         self.output.add_info('E', pkg, 'pem-private-key', f)
 
+                if tcl_regex.search(f):
+                    self.output.add_info('W', pkg, 'tcl-extension-file', f)
```
Comment (scarabeusiv) [inline rpmlint/checks/FilesCheck.py]: Warning or error?

Comment (marxin) [reply rpmlint/checks/FilesCheck.py]: Let's make it an error.

**Pull request merged**